getsolus/packages
# llvm: Add optional stage 0 bootstrap compiler

**Summary**
- For major llvm updates or flag changes `BOOTSTRAP=1` can be set to rebootstrap llvm against itself making it self-contained.
- Build with BIND_NOW enabled.
- Set BOOTSTRAP=1 for ldflags change.

**Test Plan**

- Build with `BOOTSTRAP=1` and verify stage 0 gets built and it used to build the rest of the toolchain
- Build with `BOOTSTRAP=0` and verify the system clang is used to build the toolchain and stage 0 doesn't get built.

**Checklist**
